react-components/src/components/RevealToolbar/LayersContainer/LayersContainer.tsx

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type ReactElement } from 'react';
+
+import { Menu } from '@cognite/cogs.js';
+import styled from 'styled-components';
+
+import { CadModelLayersContainer } from './CadModelLayersContainer';
+import { PointCloudLayersContainer } from './PointCloudLayersContainer';
+import { Image360CollectionLayerContainer } from './Image360LayersContainer';
+
+export const LayersContainer = (): ReactElement => {
+  return (
+    <Container>
+      <StyleMenu>
+        <CadModelLayersContainer />
+        <PointCloudLayersContainer />
+        <Image360CollectionLayerContainer />
```

Comment:
Should do something in the case when specific model type is not loaded. We can either gray out specific option and not allow to go deeper on the menu or have a fallback in that component that when nothing is present, output a  text like "No 3D models are present"

Reply:
Will connect with Kereem for his input on this & update accordingly in separate PR